chroma_core/models/host.py

```diff
@@ -1783,3 +1783,13 @@ def alert_message(self):
 
 class NoNidsPresent(Exception):
     pass
+
+
+class MountSnapshotStep(Step):
+    def run(self, kwargs):
+        self.invoke_rust_agent_expect_result(kwargs["host"], "snapshot_mount", [kwargs["fsname"], kwargs["name"]])
```

Comment:
Btw, why `kwargs['host']` was choosen to be the first parameter?

Reply:
That's the syntax of `invoke_rust_agent_expect_result` - `host` is first

Reply:
The first parameter needs to be the fqdn of the host to send the RPC to.